merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelProcessor.java

```diff
@@ -108,9 +108,10 @@ public boolean process(final Set<? extends TypeElement> annotations, final Round
     // Iterate over all elements annotated with @MissionModel
     for (final var element : roundEnv.getElementsAnnotatedWith(MissionModel.class)) {
       final var packageElement = (PackageElement) element;
+      final var missionModelParser = new MissionModelParser(elementUtils, typeUtils);
```

Comment:
This can be hoisted out of the loop, since it isn't dependent on the loop variable.

Reply:
Ope – good catch! Will do

Reply:
Done :) Thanks

Reply:
(I can't resolve threads, so you'll have to do it for me 😅)